operations/host_spawn.go

```diff
@@ -76,6 +80,22 @@ func hostCreate() cli.Command {
 				Name:  noExpireFlagName,
 				Usage: "make host never expire",
 			},
+			cli.StringSliceFlag{
+				Name:  wholeWeekdaysOffFlagName,
+				Usage: "for an unexpirable host, the days when the host should be turned off for its sleep schedule (allowed values: Sunday, Monday, Tuesday, Wednesday, Thursday, Friday, Saturday)",
+			},
+			cli.StringFlag{
+				Name:  dailyStartTimeFlagName,
+				Usage: "for an unexpirable host, the time when the host should start each day for its sleep schedule",
+			},
+			cli.StringFlag{
+				Name:  dailyStopTimeFlagName,
```

Comment:
can we add the format for these flags similar to how you added it to modify? ex `(format: HH:MM, e.g. 12:34`

Reply:
Sure, added it.